README.md

```diff
@@ -45,16 +45,16 @@ $ sudo apt-get install bazel
 * Install Visual Studio 2015 or 2017 with the Windows SDK and C++ tools
 * Install bazel via [chocolatey](https://chocolatey.org/) or the [bazel releases page](https://github.com/bazelbuild/bazel/releases):
 ```
-choco install bazel
+$ choco install bazel
```

Comment:
changed to > for Windows and kept $ on other platforms and the common steps

also, cmder defaults to λ